### PR TITLE
docs: add zh_CN po files

### PR DESCRIPTION
**Pull request by llama-factory**
## Motivation

contribute Chinese documentation

## Modifications

add Chinese po files

## Checklist

- [x] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [x] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [x] Update documentation as needed, including docstrings or example tutorials.

#1565

### CODE REVIEW

**Comment by merrymercy** [comment]
Thanks for the contribution.

Can you provide the scripts to generate these files? So that we can regenerate them next time if we update the docs.

**Comment by llama-factory** [comment]
> Thanks for the contribution.
> 
> Can you provide the scripts to generate these files? So that we can regenerate them next time if we update the docs.

ok, 
source code: https://github.com/llama-factory/llamafactory/blob/main/code/sphinx_translate.py
prompt in this [blog](https://blog.csdn.net/budahui/article/details/142748857?spm=1001.2014.3001.5502)

**Review by merrymercy — APPROVED** [review]
LGTM.  @zhyncs please take another review and we can probably merge it.

File: docs/locale/zh_CN/LC_MESSAGES/choices_methods.po [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+"Unconditional likelihood normalized selects the option with the highest "
+"average token logprob once normalized by the unconditional token logprobs, "
+"as described in [this EleutherAI "
+"blogpost](https://blog.eleuther.ai/multiple-choice-normalization/). This "
+"method incurs an additional LLM call to obtain the unconditional "
+"likelihoods."
+msgstr ""
+"无条件似然归一化选择平均令牌对数概率最高的选项，该选项通过无条件令牌对数概率进行归一化，如 [这篇 EleutherAI "
+"博客文章](https://blog.eleuther.ai/multiple-choice-normalization/) 中所述。此方法需要额外的 "
+"LLM 调用来获取无条件似然。"
```
Comment (zhyncs) [inline docs/locale/zh_CN/LC_MESSAGES/choices_methods.po:108]: This translation is too stiff.🥲

**Review by zhyncs — CHANGES_REQUESTED** [review]
@llama-factory First of all, thanks for your work! I went through this document as a whole and found that the translation of Gemini 1.5 Flash is rather inaccurate, and it seems not to have understood some technical terms in the original text. Can you manually revise it or try other APIs like GPT-4o?

**Comment by llama-factory** [comment]
> @llama-factory First of all, thanks for your work! I went through this document as a whole and found that the translation of Gemini 1.5 Flash is rather inaccurate, and it seems not to have understood some technical terms in the original text. Can you manually revise it or try other APIs like GPT-4o?

we currently don’t have the budget to call better models like GPT-4 or Claude 3.5. The source code and usage instructions have already been provided earlier. If you find it valuable, feel free to use it and test the performance on more advanced models.

**Comment by zhyncs** [comment]
> > @llama-factory First of all, thanks for your work! I went through this document as a whole and found that the translation of Gemini 1.5 Flash is rather inaccurate, and it seems not to have understood some technical terms in the original text. Can you manually revise it or try other APIs like GPT-4o?
> 
> we currently don’t have the budget to call better models like GPT-4 or Claude 3.5. The source code and usage instructions have already been provided earlier. If you find it valuable, feel free to use it and test the performance on more advanced models.

Ok Thanks! @zhaochenyang20 May you try this?

**Pull request closed without merging**